ale/transformation.py

```diff
@@ -271,3 +296,85 @@ def last_time_dependent_frame_between(self, source, destination):
                 return path[i+1], path[i], edge
 
         return None
+
+    @staticmethod
+    def extract_exact_ck_times(observStart, observEnd, targetFrame):
+        times = []
+
+        FILESIZ = 128;
+        TYPESIZ = 32;
+        SOURCESIZ = 128;
+
+        currentTime = observStart
+        timeLoaded = False
+
+        count = spice.ktotal("ck")
+        file, filtyp, source, handle = spice.kdata(0, "ck", FILESIZ, TYPESIZ, SOURCESIZ)
```

Comment:
We could do some more complex logic where we open all of the CKs and then directly build the frame chain from that, but emulating ISIS here is sufficient.